---
name: Guangdun Peng
image: images/pgd.png
role: principal-investigator
# affiliation: Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences
aliases:
  - P. Guangdun
  - P Guangdun
links:
  home-page: https://english.gibh.cas.cn/people/faculty/202409/t20240905_684956.html
  orcid: 0000-0002-8586-0637
  email: peng_guangdun@gibh.ac.cn
  googlescholar: https://scholar.google.com/citations?user=s7XddBgAAAAJ&hl=en
  
---

A major accomplishment during embryonic development is the generation of cellular diversity and orderly organization of the cell types in a basic plan that serves as the blueprint for embryogenesis and morphogenesis. This hierarchy of cell orders, i.e. the cell lineage, is a tightly controlled and well-coordinated process marked by a series of cell fate decisions during embryo development. To resolve the spatial organization and molecular architecture underpinning the cell lineage segregation in mouse embryos, we developed a method to systematically survey the spatial organization of tissue cell types, named Geo-seq. This method can be implemented to investigate the transcriptome landscape in sections of the native tissues with high efficiency and accuracy, and preserve the spatial information (Nature Protocols, 2017). Using Geo-seq, the temporal and spatial regulation for the stem cell fates commitment and specification in early mouse embryos were unraveled in unprecedented details (Developmental Cell, 2016). Our study provided high resolution digital in situ hybridization attributes, defined the molecular genealogy of germ layers and continuum of pluripotent states in real time and real space resolution. The spatial-temporal transcriptome further revealed the molecular determinants that drive lineage specification and germ layer development, and uncovered the stem cell lineage differentiation in unprecedented details (Nature, 2019). This analysis also provided de novo spatial coordinates to enable single cell positional mapping.

To fulfill the needs of high-throughput single cell sequencing and spatial transcriptomic profiling, we combined the cell barcoding and in situ positional indexing with platforms such as automatic liquid processing and microfluid. We established an automatic scRNA-seq toolkit (Auto-seq) and multi-dimensional spatial -omics technologies (mi-Step). Using Auto-seq, we comprehensively examined the single cell developmental trajectories of post-implantation embryo and retrospectively mapped the single cellâ€™s location (Nature, 2019). With the help of these new tools, we have elucidated the lineage trajectory for pluripotent cell differentiation during germ layer development in single cell resolution and characterized the progenitor cell population residing the lung epithelium (Nature Genetics, 2019). 
